src/components/Input/Input.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { v4 as uuid } from 'uuid';
+import './Input.scss';
+
+const Input = (props) => {
```

Comment:
So, we're constructing an object in the host app, passing this, and using de-structuring to get the values? Why not just pass the explicitly as we're doing in logistics-app?